srtcore/handshake.h

```diff
@@ -74,6 +74,7 @@ const int SRT_CMD_REJECT = 0, // REJECT is only a symbol for return type
       SRT_CMD_KMRSP = 4,
       SRT_CMD_SID = 5,
       SRT_CMD_CONGESTION = 6,
+      SRT_CMD_FILTER = 10,
```

Comment:
Why is SRT_CMD_FILTER out of sequence (10 and not 7)?

Reply:
Well, in short - there's also other development work ongoing in parallel and this introduces its own flags here; as I remember 7 is already taken, although not sure about 8 and 9. I think there's also at least one more branch that also introduces its own extension type, which may and need not go to the main line in future. I simply act proactively to minimize merge problems in future.

Reply:
Sorry, your work in experimental branches are not good enough reason to break the sequence in this particular PR that is scheduled to go into master. Please change SRT_CMD_FILTER to 7 and make adjustments in future PRs, when ready. Thanks.